app/uk/gov/hmrc/timetopay/arrangement/services/AuditService.scala

```diff
@@ -32,39 +32,65 @@ import scala.concurrent.ExecutionContext
 class AuditService @Inject() (auditConnector: AuditConnector)(implicit ec: ExecutionContext) {
 
   def sendSubmissionSucceededEvent(
-      taxpayer:    Taxpayer,
-      bankDetails: BankDetails,
-      schedule:    PaymentSchedule,
+      arrangement: TTPArrangement,
       auditTags: Map[String, String]
   ): Unit = {
 
     val event = makeEvent(
-      taxpayer,
-      bankDetails,
-      schedule,
+      arrangement.taxpayer,
+      arrangement.bankDetails,
+      arrangement.schedule,
       auditTags,
       Json.obj(
-        "status" -> "successfully submitted direct debit and TTP Arrangement"
+        "status" -> "successfully submitted direct debit and TTP Arrangement",
+        "directDebitInstructionReference" -> arrangement.directDebitReference,
+        "paymentPlanReference" -> arrangement.paymentPlanReference
       ))
     auditConnector.sendExtendedEvent(event)
     ()
   }
 
+  def sendArrangementQueuedEvent(
+                                  arrangement: TTPArrangement,
+                                  submissionError: SubmissionError,
+                                  workItem: TTPArrangementWorkItem,
+                                  auditTags: Map[String, String]
+                                ): Unit = {
+    val event = makeEvent(
+      arrangement.taxpayer,
+      arrangement.bankDetails,
+      arrangement.schedule,
+      auditTags,
+      Json.obj(
+        "status" -> "direct debit instruction success | TTP arrangement failed temporarily (DES server error) | Queued for retry",
```

Comment:
shouldn't the status be one of things in the `|`-delimited list?

Reply:
No, it's all of those things at the same time.

I've updated the status to `"direct debit instruction success but TTP arrangement failed temporarily (DES server error) - Queued for retry"`

Reply:
oh ok - my mind read the `|` as or. Too much coding 😆